parcels/particleset.py

```diff
@@ -464,8 +334,6 @@ def from_list(
             Optional list of initial depth values for particles. Default is 0m
         time :
             Optional list of start time values for particles. Default is fieldset.U.time[0]
-        repeatdt :
```

Comment:
Not sure it’s useful to also update the docstring already. We’ll have to put it back later anyways. OK to keep the change, but next time simply don’t touch public API docstrings until we’ve settled on API, especially when temporarily disabling functionality?

Reply:
Perhaps just revert the parameter list to how it was and leaving unused variables in there (syntax highlighting in vscode will show them as unused, which is fine for now).

I think at some point we need to start an issue to fully discuss how the execute loop of v4 looks different to v3 (perhaps the conclusion is v3 execute functions the same as v4 execute - but even then it would be good to see both sides). As part of that discussion, we can wrap in elements from https://github.com/OceanParcels/Parcels/pull/2060#discussion_r2175402689

Reply:
> Perhaps just revert the parameter list to how it was and leaving unused variables in there

Actually, I think something like:

```python
if repeatdt is not None:
     raise NotImplementedError
```
at the top of pset execute would be good. That way any tests are guaranteed not to pass through those variables, and allows us time to fully scope in #2065

Reply:
Fixed in https://github.com/OceanParcels/Parcels/pull/2060/commits/173411398775b91762126e8a6b2fc017812efd52